apps/protocol-frontend/src/utils/arrays.ts

```diff
@@ -0,0 +1,9 @@
+import { UIContribution } from '@govrn/ui-types';
+
+export const mergePages = (pages: UIContribution[][]) => {
+  let contributions = [] as UIContribution[];
+  for (const page of pages) {
+    contributions = [...contributions, ...page];
+  }
```

Comment:
This is the slowest way to merge two arrays, especially if large items, especially it would spread the same items for large nested arrays count. 

- The fastest way is to use spread once with `concat`: ```[].concat(...arrays)```.

[Playground](https://jsbench.me/o8kqzo8olz/1)